README.md

```diff
@@ -1,69 +1,50 @@
+```
 
-**The Lab Recruits Game.** It is a 3D game (a screenshot is shown above) written for the purpose of testing AI (like the AI of our iv4xr test agents). It features custom level that you can define yourself through a CSV file. Keep in mind that the game is also work in progress. More about this game can be found in its [repository](https://github.com/iv4xr-project/labrecruits).
+## Running unit tests
 
-### Deploying the demo
+We are using Gradle as the build system. To build and run unit tests, run:
 
-Build an executable of the [Lab Recruits](https://github.com/iv4xr-project/labrecruits). To do this you can clone or download the source code of this game. Open the project in Unity (note the specific version it needs) and build the executable from there.
+```
+./gradlew :cleanJvmTest :jvmTest --tests "spaceEngineers.mock.*"
+```
 
-For Mac: put the produced `LabRecruits.app` in `gym/Mac/bin`. For Windows: put the produced files including `LabRecruits.exe` in `gym/Windows/bin`.
+## Running iv4xr tests
 
-The demo classes are in `src/test/java/agents/demo`. The demos are by default non-visual (you don't literally see the game runs). Set the variable `TestSettings.USE_GRAPHICS` in the corresponding demo-class to `true` if you want it to be visual.
+The tests require Space Engineers running with the iv4XR plugin enabled.
 
-* Eclipse
 
-   This will allow you to run/modify/rerun the demo classes. Import the project into Eclipse as a **maven project**. The demo classes are in `src/test/java/agents/demo`. You can run them as junit tests.
+```
+./gradlew :cleanJvmTest :jvmTest --tests "spaceEngineers.iv4xr.*"
+```
 
-* Maven
 
-   This is if you just want to check that the project builds and that all its tests pass.
-   Just do `mvn compile` and `mvn test` at the project root.
+## Running BDD feature tests
 
-### Other documentations
+Tests scenarios also require Space Engineers running with the iv4XR plugin enabled.
 
-* For iv4xr team: [World Object Model](./docs/Observation.md)
-* For iv4xr team: [where to find goals and tactic](./docs/LRtestingLib.md)
-* For others: [basic interface to control _Lab Recruits_](./docs/BasicInterface.md)
+For now we run BDD tests from Idea.
 
-### What's in the package
-
-* `./src` the source files. It follows Maven's convention, so the root of the source files is in `src/main/java` and the root of tests' source files is in `src/test/java`.
-* `./gym` contains the binary of the _Lab Recruits_ game (macos and windows).
-* `./src/test/resources/levels` contain sample level definitions for _Lab Recruits_.
-
-
-### Contributors
-
-**Computer Science students from Utrecht University:**
-Adam Smits,
-August van Casteren,
-Bram Smit,
-Frank Hoogmoed,
-Jacco van Mourik,
-Jesse van de Berg,
-Maurin Voshol,
-Menno Klunder,
-Stijn Hinlopen,
-Tom Tanis.
-**Others:** Wishnu Prasetya, Naraenda Prasetya.
+* Right-click .feature file in Jetbrains IDEA (in src/jvmTest/resources/features) and select "Run". 
```

Comment:
Please list the plugins required to run Cucumber.

* Gherkin
* Cucumber for Java

Plus link [here](https://www.jetbrains.com/help/idea/enabling-cucumber-support-in-project.html#cucumber-plugin).